packages-exp/auth-exp/src/core/auth/initialize.ts

```diff
@@ -34,6 +36,24 @@ export function initializeAuth(
   return auth;
 }
 
+export function useEmulator(
+  authExtern: externs.Auth,
+  hostname: string,
+  port: number
+): void {
+  const auth = _castAuth(authExtern);
+  if (!auth._canInitEmulator) {
+    fail(AuthErrorCode.EMULATOR_CONFIG_FAILED, {
```

Comment:
why not assert instead of fail since you have the if block?

Reply:
Because I'm silly